client/Mac/MRDPView.m

```diff
@@ -182,54 +179,36 @@ DWORD mac_client_thread(void* param)
 		freerdp* instance = context->instance;
 		MRDPView* view = mfc->view;
 		rdpSettings* settings = context->settings;
-		
 		status = freerdp_connect(context->instance);
-		
+
 		if (!status)
 		{
 			[view setIs_connected:0];
 			return 0;
 		}
-		
+
 		[view setIs_connected:1];
-		
 		nCount = 0;
-		
 		events[nCount++] = mfc->stopEvent;
-		
-		if (settings->AsyncUpdate)
-		{
-			if (!(updateThread = CreateThread(NULL, 0, (LPTHREAD_START_ROUTINE) mac_client_update_thread, context, 0, NULL)))
-			{
-				WLog_ERR(TAG,  "failed to create async update thread");
-				goto disconnect;
-			}
-		}
-		else
-		{
-			if (!(updateEvent = freerdp_get_message_queue_event_handle(instance, FREERDP_UPDATE_MESSAGE_QUEUE)))
-			{
-				WLog_ERR(TAG, "failed to get update event handle");
-				goto disconnect;
-			}
-			events[nCount++] = updateEvent;
-		}
-		
```

Comment:
@MartinHaimberger Yes, moved to the core library (as every client needs to call that)